lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -597,23 +617,29 @@ def test_student_progress(self):
 
 
 @ddt.ddt
-class CCXCoachTabTestCase(ModuleStoreTestCase):
+class CCXCoachTabTestCase(SharedModuleStoreTestCase):
     """
     Test case for CCX coach tab.
     """
+    @classmethod
+    def setUpClass(cls):
+        super(CCXCoachTabTestCase, cls).setUpClass()
+        cls.ccx_enabled_course = CourseFactory.create(enable_ccx=True)
```

Comment:
That flag defaults to `False`, so there are only really two states for the course itself. There is a separate flag that enables CCX at the system level -- the various combinations of those is what the `ddt` combo check in this test case is exercising.